proms-resources/frontend/src/main/frontend/src/proms/ToUDialog.jsx

```diff
@@ -152,7 +167,7 @@ function ToUDialog(props) {
       { error ?
         <Alert severity="error">
           <AlertTitle>An error occurred</AlertTitle>
-          {error}
+          {error ? error : null}
```

Comment:
We already check if error is truthy 3 lines above, this change is unnecessary.
```suggestion
          {error}
```